docs/pages-app.md

```diff
@@ -0,0 +1,187 @@
+# The Pages app
+
+`cms.apps.pages` allows users to add a page tree and page content to the website. However, it does not do anything by itself; It requires you to add models into your project which inherit from `cms.apps.ContentBase`
+
+## How it works
+
+In `cms.apps.pages.models` there is a model named `Page`, which contains a set of generic page fields such as `parent`, `publication_date`, `expiry_date` and so on.
+
+To be able to add a `Page` to the site, we need a content model.  A content model is a way of representing a type of page. This could be a homepage, a standard page, a contact page etc -- each content model would have its own front-end template thus allowing you to provide different layout types, themes etc throughout your site.
```

Comment:
I find the em dash here odd. While correct it hasn't been used anywhere else. Maybe restructure to something like `A content model is a way of representing a type of page. For example, you could have a different ContentBase for your homepage, a standard page and a contact page. Each content model would have its own front-end template thus allowing you to provide different layout types, themes etc throughout your site.`